lib/components/progress_hud/progress_hud.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/foundation.dart';
```

Comment:
did you feel the folder `progress_hub` is redundant? 🤓 I think we can put common widget in a common folder, tho